src/utils/error.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * ISC License (ISC)
+ * Copyright 2018 aeternity developers

```

Comment:
```suggestion
 * Copyright 2021 aeternity developers
```
It would be more correct. Actually, I didn't see similar each-file licenses in other aeternity projects, maybe we can drop it?

Reply:
Removing it.